src/components/modal/ModalBody.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { ModalBody as ChakraModalBody, ModalBodyProps } from '@chakra-ui/react';
+
+const ModalBody: FC<PropsWithChildren<ModalBodyProps>> = (props) => {
```

Comment:
try wrapping the components in a `forwardRef`. This might solve your issue with the portal

Reply:
Not working :(